test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -229,14 +251,37 @@ int run_test_with_two_queue(cl_context &context, cl_command_queue &cmd_queue1,
 
                 if (iter == 0)
                 {
-                    vkQueue.submit(vkCommandBuffer, vkVk2CLSemaphore);
+                    if (use_fence)
+                    {
+                        vkQueue.submit(vkCommandBuffer, fence);
```

Comment:
Will need similar changes to other files especially to test_vulkan_interop_image.cpp

Reply:
So as I aready mentioned at the beginnig of the task, I don't know how it should be done for the image part. I tried changing it in this commit accordingly for bufffer and image but I ended up with deadlocks:
https://github.com/KhronosGroup/OpenCL-CTS/commit/9e83f008f5469af2e13ff4f1c7602351485d14ea#diff-2b3fe9aac0e9f06312676bdb7212bb775c8b1386f633d73db32aebb87b283f3f
Do you have any idea on how it should be implemented?

Reply:
Will take a look. We can cover image changes later if there are more issues to address.